static_frame/core/index.py

```diff
@@ -382,6 +383,15 @@ def copy(self: I) -> I:
         '''
         return self.__copy__() #type: ignore
 
+    def __sizeof__(self: I) -> int:
```

Comment:
Something to consider:
`__sizeof__` only counts the size of the container objects and not its contents for things like list.

E.g. `[2, 3, 4]` has the same size as `['abc', 'def', 'ghi']` since they're both lists with 3 elements.

Maybe we should follow this paradigm with StaticFrame containers too. If we do this, to count the full size of the container and its contents, we could add handlers in the total_getsizeof utility function..

Reply:
I think it is better to try to show true inclusive memory foot print, as that is what I think most users want (and that is pretty much what NumPy is doing). I am not sure why common containers give up at the first level... maybe for performance... but it just does not seem useful to only show one depth level of size.